src/raster/r.sun.daily/r.sun.daily.html

```diff
@@ -43,5 +43,5 @@ <h2>SEE ALSO</h2>
 
 <h2>AUTHORS</h2>
 
-Vaclav Petras, <a href="http://geospatial.ncsu.edu/osgeorel/">NCSU OSGeoREL</a>,<br>
-Anna Petrasova, <a href="http://geospatial.ncsu.edu/osgeorel/">NCSU OSGeoREL</a>
+Vaclav Petras, <a href="https://geospatial.ncsu.eduosgeorel/">NCSU OSGeoREL</a>,<br>
+Anna Petrasova, <a href="https://geospatial.ncsu.eduosgeorel/">NCSU OSGeoREL</a>
```

Comment:
Wrong URL

Reply:
(missing slash)

Reply:
While at it, you can change it to: https://geospatial.ncsu.edu/geoforall/

Reply:
Done in ac1dbb7380334ced839bfc6e5fbb4e9d86370b8d